inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/exec_graph_info.cpp

```diff
@@ -15,12 +15,21 @@ namespace {
     const std::vector<std::map<std::string, std::string>> configs = {
             {},
     };
+    const std::vector<std::map<std::string, std::string>> multiConfigs = {
+            {{ InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES,
+            std::string(CommonTestUtils::DEVICE_GPU) + "," + std::string(CommonTestUtils::DEVICE_CPU)}}
+    };
 
     INSTANTIATE_TEST_CASE_P(smoke_BehaviorTests, ExecGraphTests,
                             ::testing::Combine(
                                     ::testing::ValuesIn(netPrecisions),
                                     ::testing::Values(CommonTestUtils::DEVICE_GPU),
                                     ::testing::ValuesIn(configs)),
                             ExecGraphTests::getTestCaseName);
-
-}  // namespace
\ No newline at end of file
+    INSTANTIATE_TEST_CASE_P(smoke_Multi_BehaviorTests, ExecGraphTests,
+                            ::testing::Combine(
+                                ::testing::ValuesIn(netPrecisions),
+                                ::testing::Values(CommonTestUtils::DEVICE_MULTI),
+                                ::testing::ValuesIn(multiConfigs)),
+                            ExecGraphTests::getTestCaseName);
```

Comment:
how does it work? exec graph for MULTI case? which graph CPU or GPU is shown?

Reply:
Well, it passes OK.
But, looking closely, this happens to be the exact copy of CPU plugin test functionality, so I'll remove it.